tackle-test-generator-ui/src/org/konveyor/tackletest/ui/crawljax/CrawljaxRunner.java

```diff
@@ -178,42 +177,71 @@ private static boolean getBooleanTypeOption(TomlTable optionSpec, String optionN
     private static void handleClickablesElementSpec(TomlTable[] clickablesElemSpec, boolean dontClick,
                                                     CrawljaxConfiguration.CrawljaxConfigurationBuilder builder) {
         for (TomlTable elem : clickablesElemSpec) {
-            String tagName = elem.getString("tag_name");
+            List<String> tags = new ArrayList<>();
+            try {
+                // parse tag_name as a string
+                tags.add(elem.getString("tag_name"));
```

Comment:
What if `tag_name` doesn't exist? do we want to add a clear error message?

Reply:
Updated to throw exception on missing `tag_name` property